### PR TITLE
chore(starters): add starter intl with react-intl

### PR DESCRIPTION
**Pull request by tomekskuta**
## Description

Add Gatsby Starter Intl which creates static pages for every locale.

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1983,3 +1983,19 @@
     - Dynamic content from Livingdocs.
     - Built-in component library.
     - Robust template and theme.
+- url: https://gatsby-starter-intl.tomekskuta.pl
+  repo: https://github.com/tomekskuta/gatsby-starter-intl
+  description: Gatsby v2 i18n starter which makes static pages for every locale and detect your browsers lang. i18n with react-intl.
+  tags:
+    - i18n
+    - intl
+    - react-intl
```
Comment (wardpeet) [inline docs/starters.yml]:
```suggestion
```

File: docs/starters.yml [hunk 2]
```diff
@@ -1983,3 +1983,19 @@
     - Dynamic content from Livingdocs.
     - Built-in component library.
     - Robust template and theme.
+- url: https://gatsby-starter-intl.tomekskuta.pl
+  repo: https://github.com/tomekskuta/gatsby-starter-intl
+  description: Gatsby v2 i18n starter which makes static pages for every locale and detect your browsers lang. i18n with react-intl.
+  tags:
+    - i18n
+    - intl
```
Comment (wardpeet) [inline docs/starters.yml]: this is a bit to specific, we already have i18n

```suggestion
```

File: docs/starters.yml [hunk 3]
```diff
@@ -1983,3 +1983,19 @@
...
+    - react-intl
+    - v2
```
Comment (wardpeet) [inline docs/starters.yml]: you don't need it 😄 
```suggestion
```

File: docs/starters.yml [hunk 4]
```diff
@@ -1983,3 +1983,19 @@
...
+    - hooks
```
Comment (wardpeet) [inline docs/starters.yml]:
```suggestion
```

File: docs/starters.yml [hunk 5]
```diff
@@ -1983,3 +1983,19 @@
...
+    - jest
```
Comment (wardpeet) [inline docs/starters.yml]:
```suggestion
```

**Comment by tomekskuta** [comment]
@sidharthachatterjee @wardpeet thank you for your work! It's my first contribution to open source and it's great experience to see how it works ;)

**Pull request merged**